services/api/service.go

```diff
@@ -237,6 +238,11 @@ func NewRelayAPI(opts RelayAPIOpts) (api *RelayAPI, err error) {
 		api.ffDisableLowPrioBuilders = true
 	}
 
+	if os.Getenv("DISABLE_QUERY_PAYLOAD_ATTRIBUTES") == "1" {
```

Comment:
rather than using the term disable, i think it might be clearer to use "ENABLE_SSE_PAYLOAD_ATTRIBUTES".